solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -155,4 +176,84 @@ mod benchmark_helper {
             RuntimeOrigin::from(pallet_xcm::Origin::Xcm(location))
         }
     }
+
+    impl snowbridge_pallet_system::BenchmarkHelper<RuntimeOrigin> for () {
+        fn make_xcm_origin(location: Location) -> RuntimeOrigin {
+            RuntimeOrigin::from(pallet_xcm::Origin::Xcm(location))
```

Comment:
is this the ethereum_system pallet? If so, I dont believe this is the runtime origin you should put, maybe look at https://github.com/moondance-labs/tanssi/pull/742

Reply:
Removed it.